src/Propel/Generator/Manager/ModelManager.php

```diff
@@ -194,4 +196,33 @@ protected function doBuild(AbstractOMBuilder $builder, $overwrite = true)
 
         return 1;
     }
+
+    /**
+     * Create script to import all table map files into database map.
+     *
+     * @return int
+     */
+    protected function createTableMapLoaderScript(): int
+    {
+        $schemas = $this->getDataModels();
+        $builder = new TableMapLoaderScriptBuilder($this->getGeneratorConfig());
+        $fileContent = $builder->build($schemas);
+
+        $file = $builder->getFile();
+        $filePath = $file->getPathname();
+
+        $action = $file->isFile() ? 'Updating' : 'Creating';
+        $this->log("\t-> $action $filePath");
+
+        if ($file->isFile() && file_get_contents($filePath) === $fileContent) {
+            $this->log("\t\t(no change)");
+
+            return 0;
+        }
+        $this->log('Generating script for loading table maps at ' . $file->getRealPath());
+        $this->filesystem->mkdir($file->getPath());
+        file_put_contents($filePath, $fileContent);
+
+        return 1;
```

Comment:
Is that an error? Otherwise I would recommend using a non 1 code, e.g 2 - ideally as CODE_CHANGES class const somewhere.

Reply:
see above